base/strings/search.jl

```diff
@@ -1,5 +1,8 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+const Byte = Union{Int8,UInt8}
+const ByteArray{T<:Byte} = Vector{T}
```

Comment:
This needs to be `Union{Vector{UInt8},Vector{Int8}}`; otherwise it includes `Vector{Union{Int8,UInt8}}`.

Reply:
Ah, right. Thank you. Fixed.